src/App.css

```diff
@@ -31,5 +31,58 @@ input {
 main {
   min-height: 30rem;
```

Comment:
same - have your own css file, don't go to upper branch, put your css in your water folder